build.gradle

```diff
@@ -373,3 +373,29 @@ jmh {
     iterations = 10
     fork = 2
 }
+
+configurations {
+ xjc
+}
+ 
+dependencies {
+ xjc group: 'com.sun.xml.bind', name: 'jaxb-xjc', version: '2.2.4-1'
+}
+
+ def schemaTargetDir = new File ('src/main/gen')
+
+task createDirs () {
+ schemaTargetDir.mkdirs()
+}
+ 
+task xjc () { 
+ ant.taskdef(name: 'xjc', classname: 'com.sun.tools.xjc.XJCTask', classpath: configurations.xjc.asPath)
+ 
+ ant.xjc(
+  destdir: schemaTargetDir,
+  schema: 'src/main/resources/xjc/medline.xsd'
+ )
```

Comment:
This needs to be executed at runtime, not during configuration time. Please have a look at the gradle documentation.